speculos/mcu/seproxyhal.py

```diff
@@ -48,23 +48,61 @@ class SephTag(IntEnum):
 RENDER_METHOD = RenderMethods(0, 1)
 
 
-def ticker(add_tick):
-    """
-    Function ran in a thread to trigger a tick at a periodic interval.
+class TimeTickerDaemon(threading.Thread):
+    def __init__(self, add_tick, *args, **kwargs):
+        super(TimeTickerDaemon, self).__init__(name="time_ticker", daemon=True)
```

Comment:
```suggestion
        super().__init__(name="time_ticker", daemon=True)
```
No need to explicit the motherclass since Python3.